tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 # By default, we want to run tests for Python 2.7, Python 3.5, and run our
 # flake8 checks.
-envlist = py27,py35,flake8
+envlist = py27,py35,flake8,coveralls
```

Comment:
Do you have a good pointer for this? It was painful setting up coveralls with tox due to the state of documentation, I'm not sure how to do this without a lot of trial and error.